src/Microsoft.CodeAnalysis.Razor.Workspaces/ProjectSystem/MvcExtensibilityConfiguration.cs

```diff
@@ -28,19 +30,24 @@ internal class MvcExtensibilityConfiguration : ProjectExtensibilityConfiguration
             Kind = kind;
             RazorAssembly = razorAssembly;
             MvcAssembly = mvcAssembly;
+            RazorLanguageVersion = razorLanguageVersion;
 
             Assemblies = new[] { RazorAssembly, MvcAssembly, };
         }
 
         public override IReadOnlyList<ProjectExtensibilityAssembly> Assemblies { get; }
 
-        // MVC: '2.0.0' (fallback) or MVC: '2.1.3'
-        public override string DisplayName => $"MVC: {MvcAssembly.Identity.Version.ToString(3)}" + (Kind == ProjectExtensibilityConfigurationKind.Fallback? " (fallback)" : string.Empty);
+        // MVC: '2.0.0' (fallback) | Razor Language '2.0.0'
+        // or 
+        // MVC: '2.1.3' | Razor Language '2.1.3'
+        public override string DisplayName => $"MVC: {MvcAssembly.Identity.Version.ToString(3)}" + (Kind == ProjectExtensibilityConfigurationKind.Fallback? " (fallback)" : string.Empty) + " | " + RazorLanguageVersion;
 
         public override ProjectExtensibilityConfigurationKind Kind { get; }
 
         public override ProjectExtensibilityAssembly RazorAssembly { get; }
 
+        public override RazorLanguageVersion RazorLanguageVersion { get; }
```

Comment:
Exposing `RazorLanguageVersion` here and not `RazorConfiguration` because I'm not convinced that we should tie projects to runtime/design time. Someone else should make that decision depending on how this extensibility configuration is consumed.

Reply:
I'm not sure I understand the rationale behind that, but this seems like a safe minimal change for now. We'll need to revisit this class again in the near future, so we'll eventually do the right thing anyway 😆. Let's discuss when we get back to this spot.